config/crd/bases/cassandra.k8ssandra.io_cassandrabackups.yaml

```diff
@@ -4213,8 +4213,6 @@ spec:
               startTime:
                 format: date-time
                 type: string
-            required:
```

Comment:
Was this change intended? Not in the changelist.

Reply:
Yes it was intended. I added an explanation in the commit log. Here is what I said:

>    I have changed the CassdcTemplateSpec to a pointer. I think that makes more
    sense going forward for situations where we need to determine whether the
    property has been set. I manually tested a backup/restore operation to make
    sure that the change does not break anything.